multimodal/src/autogluon/multimodal/learners/semantic_segmentation.py

```diff
@@ -142,8 +297,11 @@ def predict(
         -------
         Array of predictions, one corresponding to each row in given dataset.
```

Comment:
Need to update the docstrings of `predict`. Returns need to tell the shape and type of predictions.